src/lib/components/PriorPosteriorDistribution/utils/prior_posterior_distribution.js

```diff
@@ -56,9 +56,14 @@ class D3PriorPosterior {
             ...this.data.values.map((values) => Math.max(...values))
         );
 
+        this.xscale_addition = (this.global_max - this.global_min) * 0.05;
```

Comment:
I'm wondering if we can avoid the magic addition constant `0.05` by trying changing this line
https://github.com/equinor/webviz-subsurface-components/blob/292791116c7afc2a8e5f24242e845b40dbdef120/src/lib/components/PriorPosteriorDistribution/utils/prior_posterior_distribution.js#L94

to something like:
```javascript
 .thresholds(d3.range(this.global.min, this.global.max, (this.global.max - this.global.min) / this.number_bins)); 
```

**Reference:** https://stackoverflow.com/questions/41396211/how-to-create-histogram-with-fixed-bin-width-in-d3-js-v4

Would that also solve the original issue? :thinking: